### ISSUE TITLE
Support slot-based Shadow DOM v1

### ISSUE THREAD

**Issue by laco0416** [webcomponents/webcomponentsjs#430]
Hi teams.
Are you planning to support Shadow DOM v1?
(Spec: http://w3c.github.io/webcomponents/spec/shadow/)

**Comment by ebidel** [comment]
Definitely! We've discussed starting this work this quarter or the next. @tjsavage for timelines.

**Comment by tomalec** [comment]
+1 for adding v1 support :)

It is mentioned in [Polymer's 2016 Roadmap](https://blog.polymer-project.org/announcements/2016/02/16/2016-roadmap/)
It would be cool to see such roadmap for webcomponents.js as well.

Just to be able plan development, and decide according to which spec version should we code. Especially in regards of vanilla Web Components.

**Comment by ebidel** [comment]
I think webcomponents.js support is implicit in supporting v1 in Polymer. cc @tjsavage @azakus @addyosmani

**Comment by tomalec** [comment]
Once v1 will be shipped, will you still polyfill v0 more or less as Hayato proposed for Blink? https://github.com/w3c/webcomponents/blob/6100444d54d8b759f456fb56fb263473494bf28c/proposals/shadow-dom-v1-in-blink.md

Will you use v0 in Chome to polyfill v1, or are you waiting for v1 to be complete in there?

**Comment by tomalec** [comment]
Is there any update on this? 

Implementation seems to be done in Chromium 
https://bugs.chromium.org/p/chromium/issues/detail?id=531990#c127
https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/zrZRD2ls5tw/ibbjD3cQAQAJ
